src/index.tsx

```diff
@@ -125,19 +110,7 @@ export function useMutation<T extends OperationBase>(
         });
       });
     },
-    [
-      resolvedEnvironment,
-      configs,
-      mutation,
-      variables,
-      uploadables,
-      onCompleted,
-      onError,
-      optimisticUpdater,
-      optimisticResponse,
-      updater,
-      isMounted,
-    ],
+    [userConfigRef, resolvedEnvironment, mutation, isMounted],
```

Comment:
```suggestion
    [resolvedEnvironment, mutation, isMounted],
```

or does eslint complain without that, and it's a no-op anyway?